src/Components/TabPanels/Account/TeamPanel.jsx

```
@@ -34,6 +34,8 @@ const TeamPanel = () => {
 	const [errors, setErrors] = useState({});
 	const [isSendingInvite, setIsSendingInvite] = useState(false);
 
+	const [getInviteToken, clearToken, isLoading, error, token] = useGetInviteToken();
```

ESLint

[error] 'isLoading' is assigned a value but never used.

(no-unused-vars)